src/game_battlealgorithm.cpp

```diff
@@ -1157,6 +1191,17 @@ int Game_BattleAlgorithm::Item::GetSourcePose() const {
 	return lcf::rpg::BattlerAnimation::Pose_Item;
 }
 
+int Game_BattleAlgorithm::Item::GetCBAMovement() const {
+	auto* source = GetSource();
+	if (source->GetType() == Game_Battler::Type_Ally && item.animation_id > 0) {
```

Comment:
Is the `animation_id > 0` check actually needed for Item, Skill and Normal?

Reply:
Did some tests myself and it seems to work without these "safeguards". Should I remove them then?

Reply:
I think so, since the `animation_id` is not used in the CBA data lookup. Also I don't see such checks in RE, but I could have missed it somewhere. The RPG_RT code for this stuff is horrible.

Reply:
Removed the `animation_id > 0` checks now.